agent/Makefile.frag

```diff
@@ -261,6 +267,13 @@ ifeq (/opt/nr/lamp/lib,$(findstring /opt/nr/lamp/lib,$(PHP_EMBED_LIBRARY)))
 	endif
 endif
 
+#
+# Need agent version for test_txn
+#
+#NEWRELIC_CFLAGS += -DNR_VERSION="\"$(AGENT_VERSION)\""
```

Comment:
The flag defining `NR_VERSION` macro is appended to `EXTRA_CFLAGS` and there should be no need for this comment:
```suggestion
```

Reply:
Good catch - fixed in [fe93413c](https://github.com/newrelic/newrelic-php-agent/pull/947/commits/fe93413c374d0700f79d6122d336f31f2c3383b1).